app/modules/munkireportinfo/views/munkireportinfo_tab.php

```diff
@@ -45,7 +45,7 @@
 			$('#munkireportinfo-reportitems').text(data.reportitems); 
             
             // Format version
-            if (data.version != "-9876543" && (data.version)) {
+            if (data.version != "-9876543" && data.version != null && (data.version)) {
```

Comment:
Just check `data.version`, null is evaluated to FALSE See http://php.net/manual/en/language.types.boolean.php